docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -71,11 +71,28 @@ implementation("io.quarkus:quarkus-webjars-locator")
 
 === From a local directory
 
-Static resources can be served from a local directory by installing an additional route in the Vert.x router.
+==== Known at build time
+
+When an application needs to configured to only service specific resources from the file system on a specific path, the easiest way to configure Quarkus is to use the `quarkus.http.static-resources.additional-local-directories` property.
+
+If for example the current working directory contains a directory named `static-resources` that contains static resources and those resources should be served under the `static` HTTP path,
+the application would be configured like so:
 
 For instance, to serve resources from the `static/` directory relative to the current path at http://localhost:8080/static/,
 you can install the following route:
 
+[source,properties]
+----
+quarkus.http.static-resources.additional-local-directories.static-resources=static
+----
+
```

Comment:
Should we add information about how the directory is resolved and what you need to do for the resources to be available at runtime when using this option?

Reply:
That's a great idea!

I'll update the PR tomorrow